src/test/java/io/strimzi/kafka/metrics/KafkaPrometheusMetricsReporterTest.java

```diff
@@ -38,31 +38,36 @@ public void testLifeCycle() throws Exception {
         KafkaPrometheusMetricsReporter reporter = new KafkaPrometheusMetricsReporter(new PrometheusRegistry());
         Map<String, String> configs = new HashMap<>();
         configs.put(PrometheusMetricsReporterConfig.LISTENER_CONFIG, "http://:0");
+        configs.put(PrometheusMetricsReporterConfig.ALLOWLIST_CONFIG, "kafka_server_group_name.*");
         reporter.configure(configs);
         reporter.contextChange(new KafkaMetricsContext("kafka.server"));
+
         Optional<Integer> port = reporter.getPort();
```

Comment:
Since we're touching this method, we can do `int port = reporter.getPort().orElseThrow();` to simplify the logic a bit.